src/QirRuntime/lib/QIR/__quantum__rt.hpp

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
For my own understanding: Which files do have a "__" naming convention?

Reply:
No convention. I wanted "__dummy.cpp" out of the way, and the headers got named after the corresponding prefixes in the QIR spec. I think the dummy is weird enough to keep as is, the headers should probably be renamed to quantum__qis.hpp and quantum__rt.hpp.